README.md

```diff
@@ -140,16 +141,22 @@ To enable a different DNS provider only when connected to the VPN you can add th
 
 ### HowTo become an administrator
 
-To add/revoke certificates, you must be allowed to decrypt `./cert/pki/private/ca.key.enc`.
-This file is encrypted with [sops](https://github.com/mozilla/sops) and your public gpg key must be added to `./.sops.yaml` by an existing administrator.
+To add/revoke certificates, you must be allowed to decrypt the sensitive encrypted files such as `./cert/pki/private/ca.key.enc`.
 
-This repository relies on [easy-rsa](https://github.com/OpenVPN/easy-rsa/blob/master/README.quickstart.md).
+These files are encrypted with [sops](https://github.com/mozilla/sops) and your public gpg key must be added to `./.sops.yaml` by an existing administrator.
+
+This repository relies on [easy-rsa](https://github.com/OpenVPN/easy-rsa/blob/master/README.quickstart.md) which is used under the hood by a custom Golang CLI wrapper named `easyvpn`.
+
+### HowTo Decrypt the Certificate Authority Key
```

Comment:
```suggestion
### HowTo decrypt the Certificate Authority key
```

Reply:
https://capitalizemytitle.com/style/Chicago/ says: `Howto Decrypt the Certificate Authority Key` (Chicago title capitalization is usually used).
(ok we might want to also update the rest of the doc)

Reply:
Not a fan... Also, not applied everywhere.
Would prefer without it, personal taste.